package.json

```diff
@@ -185,11 +185,13 @@
     },
     "patchedDependencies": {
       "babel-plugin-typescript-to-proptypes@1.4.2": "patches/babel-plugin-typescript-to-proptypes@1.4.2.patch",
-      "vite-bundle-analyzer@0.12.1": "patches/vite-bundle-analyzer@0.12.1.patch"
+      "vite-bundle-analyzer@0.16.2": "patches/vite-bundle-analyzer@0.16.2.patch",
+      "@types/webpack-env@1.18.5": "patches/@types__webpack-env@1.18.5.patch",
+      "vite@6.0.11": "patches/vite@6.0.11.patch"
```

Comment:
Some weird TS errors that I couldn't figure out...the patches just ignore those errors.